Makefile

```diff
@@ -5,7 +5,7 @@
 
 # Faz um scan em toda a app buscando strings traduzíveis e o resultado fica em app/translations/messages.pot
 make_messages:
-	pybabel extract -F config/babel.cfg -k lazy_gettext -o app/translations/messages.pot .
+	pybabel extract -F config/babel.cfg -k lazy_gettext -k __ -o app/translations/messages.pot .
```

Comment:
https://pythonhosted.org/Flask-Babel/#using-translations

*If you are using the lazy_gettext() function you should tell pybabel that it should also look for such function calls:*
```pybabel extract -F babel.cfg -k lazy_gettext -o messages.pot .```

Eso solo es para buscar el keyword ```lazy_gettext```, entonces agregue ```-k __```
También como referencia: ```pybabel extract -h```